automatic_gadget_generator/gadget.xml

```diff
@@ -122,72 +130,71 @@
                                                     '}\n'+
-                                                    'if(actionLoggerReady)\n'+
-                                                    'actionLogger.logChange(logObject);\n'+
-                                                    'var actuatorRequest = {\n'+
-                                                    'authToken: \'skfjs343kjKJ\',\n'+
-                                                    'method: \'sendActuatorData\',\n'+
-                                                    'accessRole: \'controller\',\n'+
-                                                    'actuatorId: \''+ actuator.actuatorId +'\',\n'+
-                                                    'valueNames: "",\n'+
-                                                    'data: \'\'+slideEvt.value.newValue\n'+
-                                                    '};\n'+
-                                                    'var jsonRequest = JSON.stringify(actuatorRequest);\n'+
-                                                    'if(connected)\n'+
-                                                    'ws.send(jsonRequest);\n'+
+                                                    actuator.actuatorId + 'SendData();\n'+
                                                     '});\n\n';
+                                            } else 
+                                                if (value.type=="float") {
+                                                    body+= '<p></p>'+value.name + ': &nbsp;&nbsp;<input id="' + value.name + '" type="text" data-slider-min="' + value.rangeMinimum + '" data-slider-max="' + value.rangeMaximum + '" data-slider-step="0.1" data-slider-value="0"/>\n';
+                                                    actuators+= '$("#'+ value.name +'").slider({formatter: function(currentVal){  return currentVal + "'+ value.unit +'" }});\n'+
+                                                        '$("#'+ value.name +'").on("change", function(slideEvt) {\n'+
+                                                        'var logObject = {\n'+
+                                                        '"objectType": "slider",\n'+
+                                                        '"displayName":"'+ value.name+'",\n'+
+                                                        '"content": slideEvt.value.newValue\n'+
+                                                        '}\n'+
+                                                        (graasp?'if(actionLoggerReady) actionLogger.logChange(logObject);\n':'')+
+                                                        actuator.actuatorId + 'Arr [' + valueIndex + '] = slideEvt.value.newValue;\n'+
+                                                        actuator.actuatorId + 'SendData();\n'+
+                                                        '});\n\n';
                                             }
```

Comment:
align the opening and closing brakets like this:
if (hfksjhdkfjs){

}
and same for other structures